README.md

```diff
@@ -280,6 +303,7 @@ storm::config_map:
     - class: "com.verisign.storm.metrics.GraphiteMetricsConsumer"
```

Comment:
Hmm.  I don't like `PlaintextMetricsConsumer` that much because we're not consuming plaintext metrics -- if anything, we're producing plaintext metrics (and even this is incorrect in the case of the Kafka backend, where we're emitting Avro-encoded metrics).

What about simply `MetricsConsumer` or, in case we'd like to reserve this name for future changes (like an interface), the still-fuzzy-but-slightly-better `DefaultMetricsConsumer`?